src/components/db.ts

```diff
@@ -11,6 +11,10 @@ export const openDB = async (): Promise<Database> => {
       driver: sqlite3.Database
     });
     await db.run('CREATE TABLE IF NOT EXISTS saved_data (msg_id INTEGER PRIMARY KEY,data TEXT NOT NULL);');
+    await db.run(
+      'CREATE TABLE IF NOT EXISTS suggestions (suggestion_id INTEGER PRIMARY KEY NOT NULL, suggestion_author VARCHAR(255) NOT NULL,' +
```

Comment:
I think we can shorten these column names to just `id`, `author`, `state` and so on, since given that they're in the suggestions table, we will have enough context to understand what they are